README.md

```diff
@@ -1 +1,147 @@
 # Awesome-Books-with-Modules
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 [Awesome-Books-with-Modules] <a name="about-project"></a>
+**[Awesome-Books-with-Modules]** is the first project at Microverse that is meant to test one's knowledge on gitflow and Es Linter Configuration. This project was build using Html and css. Git, Github, and Vs Code Studio were key technological tools used in this project.
+
+## 🛠 Built With <a name="built-with"> </a>
+This project is built using pure Html and css
```

Comment:
- [ ] God job Implementing ES6 syntax and modules :clap: , however please notice that we have to document our work professionally too, so that anyone who checks it out maybe a recruiter can see our good documentation skills, so I suggest you include the correct tech stacks here, you are missing javascript to be exact